redaxo/src/addons/be_style/plugins/customizer/assets/js/main.js

```diff
@@ -1,4 +1,20 @@
 $(document).on('rex:ready', function (event, container) {
+	if (container.find(rex.customizer_codemirror_selectors).size() > 0) {
```

Comment:
fehlen hier quotes?

Reply:
Die rex js property wird in der boot.php gesetzt.

Reply:
Genau die property wird gesetzt!